src/ui/profile_preferences.ui

```diff
@@ -1251,29 +1251,29 @@
              <bool>true</bool>
             </property>
             <property name="text">
-             <string/>
+             <string>ANSI [0]</string>
```

Comment:
How about we have this as tooltips? Showing ANSI colours is not something that is entirely necessary (and was not a much-requested feature), so they don't need to have that much prominence - and it looks kind of poor too:

![profile preferences_147](https://user-images.githubusercontent.com/110988/40710948-3e5128be-63fa-11e8-9b69-cdb382c7d111.png)